weldx/util.py

```diff
@@ -385,7 +315,6 @@ def to_pandas_time_index(
         pd.TimedeltaIndex,
         pd.DatetimeIndex,
         xr.DataArray,
-        "tf.LocalCoordinateSystem",
```

Comment:
I removed this type hint to make it easier, the function should be refactored into `Time` anyway